e2e/terraform/packer/ubuntu-jammy-amd64/setup.sh

```diff
@@ -132,6 +133,16 @@ sudo mv /tmp/nomad-pledge-driver ${NOMAD_PLUGIN_DIR}
 sudo mv /tmp/pledge /usr/local/bin
 sudo chmod +x /usr/local/bin/pledge
 
+# Exec2
+echo "Installing Exec2 Driver"
+sudo hc-install install --path ${NOMAD_PLUGIN_DIR} --version v0.1.0-alpha.2 nomad-driver-exec2
+sudo chmod +x ${NOMAD_PLUGIN_DIR}/nomad-driver-exec2
+
+# Envoy
+echo "Installing Envoy"
+sudo curl -s -S -L -o /opt/bin/envoy https://github.com/envoyproxy/envoy/releases/download/v1.30.1/envoy-1.30.1-linux-x86_64
+sudo chmod +x /opt/bin/envoy
+
```

Comment:
I'm guessing we probably want to only test to 1.29.x since Consul 1.19 (upcoming release) is targeting 1.29.x cc @zalimeni

Reply:
Lacking context for how Envoy testing in Nomad has been done previously, yes I'd agree that targeting the same minor version line (1.29 latest) on `<latest-release-series>` in both products is probably ideal, if there's no other reason to test further out 👍🏻

Reply:
thanks for pointing this out; i've adjusted the version to be 1.29.4